src/Microsoft.Identity.Client/Telemetry.cs

```diff
@@ -30,64 +30,65 @@
 
 namespace Microsoft.Identity.Client
 {
-    /// <summary>
-    ///     Handler enabling your application to send telemetry to your telemetry service or subscription (for instance
-    ///     Microsoft Application Insights).
-    ///     To enable telemetry in your application, you get the singleton instance of <c>Telemetry</c> by using
-    ///     <see cref="Telemetry.GetInstance()" />, you set the delegate that will
-    ///     process the telemetry events by calling <see cref="RegisterReceiver(Telemetry.Receiver)" />, and you decide if you
-    ///     want to receive telemetry
-    ///     events only in case of failure or all the time, by setting the <see cref="TelemetryOnFailureOnly" /> boolean.
-    /// </summary>
-    public class Telemetry : ITelemetryReceiver
-    {
-        /// <summary>
-        ///     Delegate telling the signature of your callbacks that will send telemetry information to your telemetry service
-        /// </summary>
-        /// <param name="events">Dictionary of key/values pair</param>
-        public delegate void Receiver(List<Dictionary<string, string>> events);
+    // TODO: obsolete this entire class.  Developer needs to implement ITelemetryReceiver and send it into the config builder to get telemetry events now.
```

Comment:
>TODO: obsolete this entire class [](start = 7, length = 32)

Given how few are likely using telemetry this may be ok. @jmprieur will surely have a comment on this